src/hyperfine/app.rs

```diff
@@ -115,6 +115,16 @@ fn build_app() -> App<'static, 'static> {
                 .short("i")
                 .help("Ignore non-zero exit codes."),
         )
+        .arg(
+            Arg::with_name("time-unit")
+                .long("time-unit")
+                .short("u")
+                .takes_value(true)
+                .value_name("millisecond|second")
```

Comment:
I would suggest to just use a value name of `UNIT` here in analogy to the other options. I would list the possible values in the `.help(..)` text: "Possible values: millisecond, second.".